grunt/config/less.js

```diff
@@ -27,7 +32,12 @@ module.exports = function (grunt, options) {
             options: {
                 baseUrl: '<%= sourcedir %>',
                 mandatory: [
-                    '<%= sourcedir %>core/less/**/*.less'
+                    '<%= sourcedir %>core/less/*.less',
+                    '<%= sourcedir %>core/less/variables/*.less',
+                    '<%= sourcedir %>core/less/global/*.less',
+                    '<%= sourcedir %>core/less/modules/*.less',
+                    '<%= sourcedir %>core/less/structure/*.less',
+                    '<%= sourcedir %>core/less/compatibility/*.less'
```

Comment:
by splitting these out into specific entries we no can't have less at "less/other/something.less" or "less/variables/other/something.less" can we revert this to **/*.less or do they need to be specified in order?